### ISSUE TITLE
[BUG] flutter_map 3.0.0, using bounds makes map not draggable anymore

### ISSUE THREAD

**Issue by Chris1234567899** [fleaflet/flutter_map#1389]
### What is the bug?

I upgraded from flutter_map 1.1.1 to 3.0.0 and all maps where I had added bounds cannot be dragged by user interactions anymore. Not sure if this is a new "feature" or a bug, but since this was working in previous versions, I assume the latter one. Or am I missing something?

### What is the expected behaviour?

According to the docs, bounds should only set the initial view, but not limit the map to move afterwards.

### How can we reproduce this issue?

```dart
Just set up a normal map und set up bounds:
FlutterMap(
          options: MapOptions(
            center: LatLng(51.509364, -0.128928),
            zoom: 9.2,
            bounds: LatLngBounds(
              LatLng(51.74920, -0.56741),
              LatLng(51.25709, 0.34018),
            ),
            keepAlive: true,
          ),
          children: [
            TileLayer(
              urlTemplate: "https://api.mapbox.com/styles/v1/mapbox/satellite-streets-v11/tiles/256/{z}/{x}/{y}?access_token=${dotenv.get('MAPBOX_TOKEN')}", subdomains: const ['a', 'b', 'c'],
            ),
          ],
        )
```

### Can you provide any other information?

I have also some additional plugins, but I assume it's the core flutter_map library.
pubspec.yaml:

  flutter_map: ^3.0.0
  flutter_map_line_editor: ^4.0.2
  flutter_map_tile_caching: ^6.1.0

### Platforms Affected

Android

### Severity

Obtrusive: Prevents normal functioning but causes no errors in the console

### Frequency

Consistently: Always occurs at the same time and location

### Requirements

- [X] I agree to follow this project's [Code of Conduct](https://github.com/fleaflet/flutter_map/blob/master/CODE_OF_CONDUCT.md)
- [X] My Flutter/Dart installation is unaltered, and `flutter doctor` finds no relevant issues
- [X] I am using the [latest stable version](https://pub.dev/packages/flutter_map) of this package
- [X] I have checked the FAQs section on the documentation website
- [X] I have checked for similar issues which may be duplicates

**Comment by AndreiMisiukevich** [comment]
@Chris1234567899 I faced the same issue and worked around it

```dart
final mapController = MapControllerImpl();

  @override
  void initState() {
    WidgetsBinding.instance.addPostFrameCallback((_) {
      if (mounted) _fitMapBounds();
    });
    super.initState();
  }

  @override
  void dispose() {
    mapController.dispose();
    super.dispose();
  }

  @override
  void didUpdateWidget(covariant ZipCodeMap oldWidget) {
    super.didUpdateWidget(oldWidget);
    _fitMapBounds();
  }
  
  Widget build(BuildContext context) {
  // use mapController inside your FlutterMap
  ...
  }
  
 void _fitMapBounds() {
 // Pass your bounds here
    mapController.fitBounds(
      LatLngBounds(...),
      options: const FitBoundsOptions(padding: EdgeInsets.all(20)),
    );
  }
  ```

**Comment by JaffaKetchup** [comment]
Duplicate of #1362, which has been fixed by #1369. Please await release (join the Discord server for notifications), or use the main branch for now. Thanks for the issue!
